plugins/woocommerce/src/StoreApi/Schemas/V1/CheckoutSchema.php

```diff
@@ -336,7 +336,7 @@ function ( $option ) {
 					},
 					$field['options']
 				);
-				if ( true !== $field['required'] ) {
+				if ( true !== $field['required'] || $this->additional_fields_controller->is_conditional_field( $field ) ) {
```

Comment:
Basically also allow an `""` for conditional selects, this allows something to be hidden and unset.